File: app/src/main/java/c/m/storyapp/register/data/mapper/RegisterMapper.kt
package c.m.storyapp.register.data.mapper

import c.m.storyapp.register.data.remote.dto.RegisterResponseDTO
import c.m.storyapp.register.domain.model.RegisterResponse


fun RegisterResponseDTO.toRegisterResponse(): RegisterResponse {
    return RegisterResponse(
        error = error,
        message = message,
    )
}

fun RegisterResponse.toRegisterResponseDTO(): RegisterResponseDTO {
    return RegisterResponseDTO(
        error = error,
        message = message,
    )
}

File: lib/register/data/mapper/register_mapper.dart
import 'package:story_app/register/data/remote/dto/register_response_dto.dart';
import 'package:story_app/register/domain/model/register_response.dart';

extension RegisterResultDTOExtension on RegisterResponseDTO {
  RegisterResponse toRegisterResponse() => RegisterResponse(
        error: error,
        message: message,
      );
}
